clients/desktop/src/vault/reshare/secure/SecureMigrateVaultPage.tsx

```
@@ -0,0 +1,100 @@
+] as const
+
+export const SecureMigrateVaultPage = () => {
+  const vault = useCurrentVault()
+  const { local_party_id, hex_chain_code, lib_type } = vault
+
+  const { step, setStep, toPreviousStep, toNextStep } = useStepNavigation({
+    steps: reshareVaultSteps,
+    onExit: useNavigateBack(),
+  })
+
+  const { t } = useTranslation()
```

ESLint

[error] 't' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)